library/extractor/src/main/java/com/google/android/exoplayer2/metadata/id3/TextInformationFrame.java

```diff
@@ -19,52 +19,77 @@
 
 import android.os.Parcel;
 import android.os.Parcelable;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 import com.google.android.exoplayer2.MediaMetadata;
 import com.google.android.exoplayer2.util.Util;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 /** Text information ID3 frame. */
 public final class TextInformationFrame extends Id3Frame {
+  private final static String MULTI_VALUE_DELIMITER = ", ";
 
   @Nullable public final String description;
+
+  /** @deprecated Use {@code values} instead. */
+  @Deprecated
   public final String value;
 
-  public TextInformationFrame(String id, @Nullable String description, String value) {
+  @NonNull
+  public final String[] values;
+
+  public TextInformationFrame(String id, @Nullable String description, @NonNull String[] values) {
     super(id);
     this.description = description;
-    this.value = value;
+    this.values = values;
+
+    if (values.length > 0) {
+      this.value = values[0];
+    } else {
+      this.value = null;
```

Comment:
I think this should `""` to be backwards compatible with the previous behaviour (please add a test for this)

Reply:
Resolved in 157f1f0.